### PR TITLE
Enhancement: DocumentDB

### PR DESCRIPTION
**Pull request by 3vilsaint**
Add support for the AWS DocumentDB service.

### CODE REVIEW

File: ScoutSuite/providers/aws/metadata.json [hunk 1]
```diff
@@ -313,6 +313,13 @@
                 }
             }
         },
+        "docdb": {
+            "resources": {
+                "clusters": {
```
Comment (j4v) [inline ScoutSuite/providers/aws/metadata.json]: I've added this so it now shows in the HTML report, but the path is wrong as it should include `.regions.id.`.

File: ScoutSuite/utils.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
     'awslambda': 'Lambda',
     'redshift': 'RedShift',
     'route53': 'Route53',
+    'docdb': 'DocumentDB',
```
Comment (j4v) [inline ScoutSuite/utils.py]: Added this to also format CLI output.

**Comment by j4v** [comment]
@3vilsaint I've made some minor changes and added a few comments to this PR.

One general comment is to decide whether you'll use `docdb` or `documentdb` and stick to it everywhere.

**Comment by klauern** [comment]
It looks like `docdb` is the canonical name used for IAM-related stuff: https://docs.aws.amazon.com/documentdb/latest/developerguide/service-linked-roles.html

**Comment by j4v** [comment]
@klauern this branch/PR is mostly abandoned I believe. Feel free to pick it up if it's of interest to you guys!

**Pull request merged**